rusk/src/lib/chain/rusk.rs

```diff
@@ -494,16 +565,43 @@ fn accept(
 fn execute(
```

Comment:
Please update the comment above to reflect the changes.

Reply:
Done